### PR TITLE
Logger and remote server display

### PR DESCRIPTION
**Pull request by Cesium1289**
The logging functionality and ability to "ls -r" should work.

### CODE REVIEW

File: Client/Connection.cs [hunk 1]
```diff
@@ -15,17 +22,28 @@ public static int Connect(ref FtpClient client, Commands.Connect commands)
...
+                    Console.WriteLine(ex.Message);
+                return 0;
             }
 
             if (client.IsAuthenticated)
-                return 0;
+            {
+                logger = new Logger(client.Credentials.UserName);
+                Console.WriteLine("Successfully connected to server!");
+                return 1;
```
Comment (MattMoradi) [inline Client/Connection.cs]: Success should return `0` and failure should return `-1`

File: Client/Get.cs [hunk 2]
```diff
@@ -21,50 +21,74 @@ public static int MultipleFiles(IEnumerable<string> files)
...
-                    Console.WriteLine("{0}/", i.Name);
-                }
+                    if (directory.Local != null)
+                    {
+                        filePath = Path.GetFullPath(directory.Local);
+                    }
+                    else if (directory.Remote != null)
+                    {
+                        throw new InvalidOperationException("listing remote files (ls -r) not implemented");         // to be implemented!
+                    }
```
Comment (MattMoradi) [inline Client/Get.cs]: Can this clause be removed now?

**Review by MattMoradi — CHANGES_REQUESTED** [review]
Mostly works. Specifying paths to the `--remote` flag doesn't work however. We could potentially implement the `cd` feature instead if you're in favor of that. Also, when listing items on the remote server it may be worth denoting subdirectories with a `/` at the end as `--local` does for consistency.

File: Client/Get.cs (hunk 2 above)
Comment (Cesium1289) [reply Client/Get.cs]: Probably. That local listing was someone else's task, so I did not want to mess with their function.

**Review by MattMoradi — APPROVED** [review]

**Pull request merged**